Makefile

```diff
@@ -43,3 +43,9 @@ install-test-requirements: ## install requirements for testing
 
 install-deploy-requirements:  ## install requirements for deployment
 	pip install -r requirements/deploy.txt
+
+install-docs-requirements:  ## install requirements for documentation
```

Comment:
Would it be worth to have just one command `install-dev-requirements` which installs the `docs.txt` and `test.txt` requirements files?

Reply:
Yeah it would be nice, but In my opinion it should be another PR.